README.md

```diff
@@ -206,6 +206,10 @@ default_brightness:
   description: default brightness for xpwm, xpwm_rgb and xpwm_rgbw only for turn on command (must be between 1 and 255)
   required: false
   type: int
+invert:
+  description: invert the value returned for binary_sensors (on become off and vice versa)
+  required: false
+  type: bool
```

Comment:
add a default to False, to assure retrocompat

Reply:
Not setting the value in `configuration.yaml` is equivalent to setting it to false. I'll try to find how to add a default

Reply:
```suggestion
  type: bool
  default: false
```